src/services/models/configModels.js

```diff
@@ -2,11 +2,10 @@
 
 import { reshape } from 'utilities';
 
-export const toDeviceGroupsModel = (response = {}) => (response.items || []).map(toDeviceGroupModel);
-
-export const toDeviceGroupModel = (device = {}) => reshape(device, {
-  'id': 'id',
-  'displayName': 'displayName',
-  'conditions': 'conditions',
-  'eTag': 'eTag'
-});
+export const toDeviceGroupsModel = (response = {}) => (response.items || [])
```

Comment:
So by changing this pattern, you're thinking we'll never need one at a time... always a list?

Reply:
Just simplifying since we only use it this way right now. If we have a call later on that does shared a model we can break that one out then.